docs/deployment/connect-scm.md

```diff
@@ -22,15 +22,32 @@ Linking a source code manager provides the following benefits:
 - For GitHub users:
     - Provides Semgrep access to post PR or MR comments.
     - For GitHub Actions users: Enables you to add a Semgrep CI job to repositories in bulk.
+- Allows you to scan and manage your Bitbucket projects in Semgrep AppSec Platform.
 
-You can only connect your Semgrep organization to the source code manager that you originally logged in with. If your organization uses both GitHub and GitLab to manage source code, log in with the source code manager that you would prefer to use to manage Semgrep org membership. You can still scan repositories from other sources.
+If your organization uses both GitHub and GitLab to manage source code, log in with the source code manager that you would prefer to use to manage Semgrep org membership. You can still scan repositories from other sources, including Bitbucket, though you will need to use SSO to manage the authentication of your users in such cases.
 
 The process to connect a source code manager depends on whether your SCM tool is cloud-hosted by the service provider, hosted on-premise, or hosted as a single tenant by the service provider.
 
 ## Connect to cloud-hosted orgs
 
 If you opted to scan a GitHub or GitLab repository when you initially signed in, you may have already performed these steps and can skip to [Next steps](#next-steps).
 
+### Bitbucket Cloud
+
+1. Sign in to Semgrep AppSec Platform.
+1. On the sidebar, click **the organization account** you want to make a connection for.
```

Comment:
```suggestion
1. On the sidebar, click **the organization account** you want to make a connection for.
```
Do you mean the account name? Can you describe the UI as how the user would see it? I think this should be broken down into 2 steps, click the account name drop down, then click whichever account you want to make a connection for.